easybuild/easyconfigs/c/ClustalW2/ClustalW2-2.1-intel-2017b.eb

```diff
@@ -0,0 +1,33 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'ClustalW2'
+version = '2.1'
+
+homepage = 'http://www.ebi.ac.uk/Tools/msa/clustalw2/'
+description = """ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['ftp://ftp.ebi.ac.uk/pub/software/%s/%s' % (name.lower(), version)]
```

Comment:
any reason not to use `%(namelower)s` and friends?

Reply:
Will fix, thanks.